### DISCUSSION TITLE
nginx代理问题

### DISCUSSION THREAD

**Discussion by blockchain315** [hhyo/Archery#2853] [Q&A]
使用docker-compose运行的archery v1.11.3服务，通过ip加端口访问正常，在外层又安装了nginx代理到9123端口访问域名不通且会自动加上9123端口，在settings.py csrf中也增加了相关的域名，这个问题我看issues中好多人碰到但未给出解决方法

**Comment by blockchain315** [comment]
请社区大佬给个解决方法

**Comment by dxq6888** [comment]
大佬们我有同样的问题解决了么？
